README.md

```diff
@@ -24,9 +24,9 @@ That's it, you are ready to use Ottoman.
 
 Supported version are:
 
-| Ottoman     | Nodejs      | Couchbase SDK   | Couchbase Server
-| ----------- | ----------- | --------------- | -----------------
-| ^2.0.0      | ^8.0.0      |  ^3.2.2         | ^6.5.0
+| Ottoman     | Nodejs  | Couchbase SDK | Couchbase Server
+| ----------- |---------|---------------| -----------------
+| ^2.0.0      | ^12.0.0 | ^4.1.3        | ^6.5.0
```

Comment:
The Couchbase SDK should still be 3.2.2, right?

Reply:
right and fixed